python/ray/data/_internal/execution/operators/output_splitter.py

```diff
@@ -47,6 +48,8 @@ def __init__(
         self._output_queue: deque[RefBundle] = deque()
         # The number of rows output to each output split so far.
         self._num_output: List[int] = [0 for _ in range(n)]
+        # The time of the overhead for the output splitter
```

Comment:
Let's also note in the comment that this is measured at operator level (as opposed to Dataset level)